frontend/packages/ux-editor/src/components/config/editModal/EditCodeList.tsx

```diff
@@ -16,11 +24,17 @@ export function EditCodeList({ component, handleComponentChange }: IGenericEditC
 
   return (
     <div>
-      <TextField
-        id='modal-properties-code-list-id'
+      <Select
+        options={
+          Array.isArray(optionList)
```

Comment:
I stedet for å sjekke at dette er et array, kan du f.eks. bruke `isLoading` og `error` fra `useOptionListQuery` til å sjekke status på spørringen og returnere en spinner mens man venter på svar eller en feilmelding hvis noe er feil. Deretter vet du at `optionList` har en gyldig verdi. Her er et eksempel: https://tanstack.com/query/v4/docs/react/examples/react/basic

Jeg ville for øvrig gitt variabelen et litt mer beskrivende navn, f.eks. `optionListIds`. Det gjelder også hooken (den kan eksempelvis hete `useOptionListIdsQuery`). Da er det tydelig at det vi henter er en liste med ID-er for kodelister.